crates/libcontainer/src/process/container_init_process.rs

```diff
@@ -204,7 +204,7 @@ pub fn container_init_process(
         // before pivot_root is called. This runs in the container namespaces.
         if let Some(hooks) = hooks {
             hooks::run_hooks(hooks.create_container().as_ref(), container)
-                .context("Failed to run create container hooks")?;
+                .context("failed to run create container hooks")?;
```

Comment:
We don't need this since these will be replaced with `thiserror`. Same for the others as well.

Reply:
Prioritize the merge for subsequent work 🙇